R/estimate_cuminc.R

```diff
@@ -38,6 +38,8 @@ estimate_cuminc <- function(data
                             ,...) {
   # check for installation of tidycmprsk package
   rlang::check_installed("tidycmprsk", version = "0.1.1")
+  dots <- rlang::dots_list(...)
+  estimate_cuminc_args <- as.list(environment())
```

Comment:
I would implement the labels in a similar way here.

Reply:
this line was left over from the last round of review. I removed the line that exports this object with the result, but forgot to delete the line created the object